tests/pages/services/QuotaOverview-cy.js

```diff
@@ -39,118 +39,204 @@ describe("Quota Tab", function() {
       cy.visitUrl({ url: "/services/quota" });
     });
 
-    function getFirstRowName(name) {
-      cy.get(".ReactVirtualized__Grid")
-        .eq(2)
-        .children()
-        .first()
-        .children()
-        .first()
-        .contains(name);
-    }
-
-    function getSecondRowName(name) {
-      cy.get(".ReactVirtualized__Grid")
-        .eq(2)
-        .children()
-        .first()
-        .children()
-        .last()
-        .contains(name);
-    }
-
-    function clickHeading(name) {
-      cy.get(".TopRightGrid_ScrollWrapper")
-        .contains(name)
-        .click();
-    }
+    context("Quota table", function() {
+      function getFirstRowName(name) {
```

Comment:
A good spike day task would be to implement these as Cypress commands in `tests/_support` similar to some of the existing form commands we use for service modal tests etc. Not in scope for this ticket though.

Reply:
I like the idea.